scss/layout/_grid.scss

```diff
@@ -138,16 +149,19 @@
     }
 
     .grid--2-wide {
-      .grid__item:nth-last-child(-n+2) {
-        border-bottom: none;
-        margin-bottom: 0;
+      .grid__item:nth-child(1),
```

Comment:
I just realized that we could use the `nth-child(-n+2)` rule to grab the first two child elements, and ``nth-child(-n+3)` for three, etc.